bin/rucio

```diff
@@ -2091,6 +2091,18 @@ def update_rule(args):
     return SUCCESS
 
 
+@exception_handler
+def move_rule(args):
+    """
+    %(prog)s move-rule [options] <ruleid> <rse_expression>
+
+    Update a rule.
+    """
+    client = get_client(args)
+    print client.move_replication_rule(rule_id=args.rule_id, rse_expression=args.rse_expression)
```

Comment:
print vs. logging ?

Reply:
For now we always print the output on the command line; If we want to do that through logging, I would say we should consistently change this everywhere in the CLI?